aspnet-core/xml/ns-Microsoft.AspNetCore.ResponseCompression.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.ResponseCompression">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types used to dynamically compress response bodies and save bandwidth.</summary>
```

Comment:
```suggestion
    <summary>Contains types used to dynamically compress response bodies and reduce bandwidth.</summary>
```

Reply:
That's a bit confusing, you're not reducing the bandwidth, you're reducing the bandwidth utilization.
```suggestion
    <summary>Contains types used to dynamically compress response bodies and reduce bandwidth utilization.</summary>
```